pos_lot_selection/__manifest__.py

```diff
@@ -6,6 +6,7 @@
     'version': '11.0.1.0.0',
     'category': 'Point of Sale',
     'author': 'Tecnativa,'
+              'Lambda IS,'
```

Comment:
Sorry, but this change doesn't grant you co-authorship in the module. Please add yourself only in contributors.

Reply:
Sure. Just curious, what guidelines do you follow on what qualifies as a change for authorship?

Reply:
There are no objective metrics, but a matter of sense about the amount of time/code involved around the module itself. In this case, it's very thankfully that you have fixed a problem in a module that is not yours, but the module essence and the amount of work invested for developing it is not something that you have needed to invest. Don't you think?

Reply:
My opinion is that anyone who modifies a file is a co-author and has copyright on those files. If order is preserved (top to bottom) it can be seen who is original author. Qualifying based on amount of work done is quite subjective and if it will be used as a criteria we should define it in the [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst), example _only improvements grant co-authorship and can be listed in the manifest, authors of bugfixes and other changes can be added in contributors_. I'll remove myself from the authors now although I did spend a surprising amount of time debugging and researching this :wink: .

Reply:
One think is the copyright of the file, and another the module co-authorship. If you have a proposal for "objectivitizing" the module co-authorship in the guidelines, please do it. I'm all opened for avoiding these misunderstandings.

Reply:
OK, I'll see if I can draft a proposal. Thanks for the review.